tests/test_alerts.py

```diff
@@ -25,6 +27,30 @@ def test_from_yaml_loads_data(tmp_path, alert_dict):
     assert isinstance(alerts.last_updated_date, AlertDate)
 
 
+@freeze_time(datetime(
+    2021, 4, 21, 11, 30, tzinfo=pytz.utc
+))
+def test_last_updated(alert_dict):
```

Comment:
Minor: could we test as much with just two alerts?

Reply:
Changed in 310dbca